README.md

```diff
@@ -145,6 +156,174 @@ readData(dataInput) {
 }
 ```
 
+# Setting Up Client Network
+
+### 7. Enabling Client TLS/SSL
+
+You can use TLS/SSL to secure the connection between the client and members. If you want TLS/SSL enabled
+for the client-cluster connection, you should set an SSL configuration. Once set, the connection (socket) is
+established out of an `options` object supplied by the user. Please see [TLS/SSL section](#1-tlsssl).
+
+As explained in the [TLS/SSL section](#1-tlsssl), Hazelcast members have key stores used to identify themselves (to other members) and Hazelcast Node.js clients have certificate authorities used to define which members they can trust. Starting with Hazelcast 3.8.1 release, mutual authentication is introduced. This allows the Node.js clients also to have their private keys and public certificates and members to have their certificate authorities so that the members can know which clients they can trust. Please see the [Mutual Authentication section](#13-mutual-authentication).
+
+
+# Securing Client Connection
+
+## 1. Supported Security Protocols
+
+This chapter describes the security features of Hazelcast Node.js Client. These features allow you to perform security activities, such as TLS/SSL and its sub-section Mutual Authentication. All of the Security features explained in this chapter are the features of **Hazelcast IMDG Enterprise** edition.
+
+### 1. TLS/SSL
+
+One of the offers of Hazelcast is the TLS/SSL protocol which you can use to establish an encrypted communication across your cluster with key stores and trust stores.
+
+- A Java `keyStore` is a file that includes a private key and a public certificate. The equivalent of a key store is the combination of `key.pem` and `cert.pem` files with pem extension at the Node.js client side.
+
+- A Java `trustStore` is a file that includes a list of certificates trusted by your application which is named certificate authority. The equivalent of a trust store is a `ca.pem` file with pem extension at the Node.js client side.
+
+#### 1.1. TLS/SSL for Hazelcast Members
+
+Hazelcast allows you to encrypt socket level communication between Hazelcast members and between Hazelcast clients and members, for end to end encryption. To use it, see [TLS/SSL for Hazelcast Members section](http://docs.hazelcast.org/docs/3.10.4/manual/html-single/index.html#tls-ssl-for-hazelcast-members).
```

Comment:
The first sentence looks like it would be a better fit for the introduction part (`###TLS/SSL`). I think we do not need "TLS/SSL for Hazelcast Members" and "TLS/SSL for Hazelcast Node.js Clients" headings. We do not care about the encryption between members. We care about the encryption between member and client. You already started talking about truststores and stuff before. Why don't you put here a paragraph or two saying "in order to use ssl, you need to configure your keystore on the server side. this is how you do it. then you need to configure your list of trusted certificates on the client side. this is how you do it. by the way you can configure any option stated in node.js ssl documentation by simply adding it to our 'options' object."

Reply:
ok, i read the rest and this part is explained later ` then you need to configure your list of trusted certificates on the client side. this is how you do it. by the way you can configure any option stated in node.js ssl documentation by simply adding it to our 'options' object.`. I am fine with that but we still need a parahraph to show how to set keystore and truststore for the member

Reply:
`To use it, see [TLS/SSL for Hazelcast Members section](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#tls-ssl-for-hazelcast-members).`
 This guides the user to the Reference Manual. He can see how to set `keyStore` and `trustStore` there. 

Also, I will add `You should set keyStore and trustStore before starting the members. See the next section how to set keyStore and trustStore on the server side.` before `TLS/SSL for Hazelcast Members` heading.